styles/style.css

```diff
@@ -0,0 +1,95 @@
+.container {
+  background-color: #e5e5e5;
+  width: 389px;
+}
+
+nav {
+  width: 20.188rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 1.5rem;
+}
+
+.flex {
+  display: flex;
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
+  background-color: #fff;
+  background-image: url("../img/header-shapes-mobile.png");
+  background-repeat: no-repeat;
+  margin: 1.625rem 0;
+  width: 28.875rem;
+  height: 36.697rem;
+  border-bottom-left-radius: 80px;
+}
+
+h1 {
+  height: 9.375rem;
+  width: 15rem;
+  margin: 0.75rem 1.5rem;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+header p {
+  width: 20.438rem;
+  margin: 0 1.5rem;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+
+  /* or 150% */
+
+  color: #344563;
+}
+
+h5 {
```

Comment:
[Optional]
My recommendation is, to use CSS classes to style the HTML elements instead of directly styling the HTML elements. Because CSS classes are reusable and easy to maintain. 😃